test/suites/image.sh

```diff
@@ -297,21 +297,32 @@ run_images_public() {
   query /1.0/images?recursion=1\&project=default | jq --exit-status '(.metadata | length) == 0 and .status_code == 200'
```

Comment:
Those tests above are making sure that every kind of auth will seen an empty list with a 200 when there are not public images.

Would it be worth adding a check that they can all list images that are indeed public? Maybe that's already done elsewhere but couldn't find it quickly.